index.html

```diff
@@ -8,6 +8,75 @@
     <link rel="stylesheet" href="./styles.css" />
   </head>
   <body>
+    <main>
+      <div class="container">
+        <section class="primary">
+          <h1 class="page-title">Contrast Ratio Calculator</h1>
+          <div class="row">
+            <div class="input-fields">
+              <div class="form-item">
+                <label for="foreground">Foreground color</label>
+              </div>
+              <div class="input-items">
+                <div class="swatch-one"></div>
+                <input id="foreground" name="foreground" class="form-input" type="text" value="#000000" />
+              </div>
+            </div>
+            <div class="input-fields">
+              <div class="form-item">
+                <label for="background">Background color</label>
+              </div>
+              <div class="input-items">
+                <div class="swatch-two"></div>
+                <input id="background" name="background" class="form-input" type="text" placeholder="Second color" />
+              </div>
+            </div>
+          </div>
+          <div class="result-row">
+            <div class="ratio-result">
+              <div>
+                <p>Contrast Ratio</p>
+                <p class="large">4.51</p>
+              </div>
+            </div>
+            <div class="check-result">
+              <div>
+                <p>Small Text</p>
+                <div class="pass">Icon</div>
+                <div class="pass">Icon</div>
+              </div>
+            </div>
+            <div class="check-result">
+              <div>
+                <p>Large Text</p>
+                <div class="fail">Icon</div>
+                <div class="fail">Icon</div>
+              </div>
+            </div>
+            <div class="check-result">
+              <div>
+                <p>Graphics</p>
+                <div class="pass">Icon</div>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <section class="secondary">
```

Comment:
We will have a seperate ticket to add the actual content here. 
But for now please add this comment 

```suggestion
<!--TODO: replace lorem ipsum with actual content-->
        <section class="secondary">
```

Reply:
I made those changes except for one. I figured we would use JS for the year but thought I would just add it manually. But do you want me to remove the paragraph tags and just have an empty section.secondary? I'll assume that is what you want and make the change but I won't push until I hear back. BTW, I can just do another push from the same branch, correct?

Reply:
Yeah, you can push up those changes.

Whenever you are asked to make changes, feel free to push those changes up and then request a re-review
<img width="356" alt="Screen Shot 2022-08-07 at 1 12 11 PM" src="https://user-images.githubusercontent.com/67210629/183309366-1f2e3677-9eca-47ef-8bdc-fb833d643aaa.png">

Then the original reviewer will take another look and approve it if everything looks good. 

> But do you want me to remove the paragraph tags and just have an empty section.secondary?

You don't need to remove anything. You can leave that there. 
Just add that comment, and we will create some tickets for people to build off of this template in future PR's. 

> Also, I added CSS for small screens so it looks "okay" on mobile for the time being.

Yeah that is fine. 
We will build from it in later PR's